package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@s-ui/tealium-tracker",
+  "version": "1.0.0",
+  "description": "Tool to track tealium tags and handle events with tealium",
+  "main": "lib/",
+  "scripts": {
+    "lib": "npm run lib:js",
+    "lib:js": "rm -Rf ./lib && mkdir -p ./lib && babel --presets sui ./src --out-dir ./lib",
+    "lint": "sui-lint js && sui-lint sass",
+    "test": "NODE_ENV=test npm run test:client && NODE_ENV=test npm run test:server",
+    "test:ci": "npm run test:client -- --ci",
+    "test:client": "NODE_ENV=test sui-test browser -P 'test/**/*Spec.js' -I 'test/server/*Spec.js'",
+    "test:client:watch": "npm run test:client -- --watch",
+    "test:server": "NODE_ENV=test sui-test server -P './test/!(browser)/*.js'",
+    "test:server:watch": "npm run test:server -- --watch",
+    "commitmsg": "validate-commit-msg",
+    "precommit": "sui-precommit run"
+  },
+  "keywords": [
+    "data-rel",
+    "track",
+    "service",
+    "tealium"
+  ],
+  "author": "Enablers frontend",
+  "license": "ISC",
+  "dependencies": {
+    "@s-ui/js": "1"
+  },
+  "devDependencies": {
+    "@s-ui/lint": "2",
+    "@s-ui/mono": "1",
+    "@s-ui/precommit": "2",
+    "@s-ui/test": "1",
+    "babel-cli": "6.24.1",
+    "babel-preset-sui": "1",
+    "chai": "4.1.2",
+    "husky": "0.13.4",
+    "sinon": "4.4.2",
```

Comment:
We have the same deps here:
https://github.com/SUI-Components/sui/blob/master/packages/sui-mockmock/package.json#L16-L17

For me we could move nock and sinon to a `sui-test-peer-dependencies` like this one https://github.com/SUI-Components/sui/tree/master/packages/sui-component-peer-dependencies

Reply:
I am totally agree with that as these are common used libraries

Reply:
Done here -> https://github.com/SUI-Components/sui/pull/238 :eyes: